core/Mail.php

```diff
@@ -99,15 +142,42 @@ public function setReplyTo($email, $name = null)
      */
     private function initSmtpTransport()
     {
-        $tr = StaticContainer::get('Zend_Mail_Transport_Abstract');
-        if (empty($tr)) {
+        $mailConfig = Config::getInstance()->mail;
+
+        if (empty($mailConfig['host'])
+            || $mailConfig['transport'] != 'smtp'
+        ) {
             return;
         }
 
-        Mail::setDefaultTransport($tr);
+        $this->isSMTP();
```

Comment:
Would it be possible, to also use PHPMailer for delivering E-Mails without SMTP? That way we could also make sure that the endless amounts of edge-cases that can occur with mail() are handled properly

Reply:
That should be the case I guess, but it's not needed to set smtp config if it's not using smtp

Reply:
Thanks for the info. It seems like I misread the function then.